_plugins/search-indexer.rb

```diff
@@ -0,0 +1,139 @@
+# frozen_string_literal: true
```

Comment:
Where's this code from? Written from scratch? Add a license header?

Reply:
good catch @dblock. @AMoo-Miki I think this is original so it needs a headering.

Reply:
It is from scratch; will add header.